client/src/routes/game/planets/id/resource_bar.rs

```diff
@@ -53,8 +53,15 @@ impl ResourceConfig {
     }
 
     pub fn produced(&self, planet: Planet, ticks: usize) -> f64 {
-      // TODO: update this when resources will be updated
-        self.producers.iter().map(|building_type| building_type.produced(planet.building_level(building_type.clone()), ticks).metal).sum::<f64>()
+        // TODO: update this when resources will be updated
+        self.producers
+            .iter()
+            .map(|building_type| {
+                building_type
+                    .produced(planet.building_level(building_type.clone()), ticks)
+                    .metal
+            })
+            .sum::<f64>()
```

Comment:
Is this supposed to be removed ?

Reply:
it will be fixed in the big resource update